pallenec

```diff
@@ -63,15 +64,42 @@ local inspect_opts = {
     end
 }
 
+local function print_tokens(lexer)
+    while true do
+        local token = lexer:next()
+        if token.name == "EOF" then
+            break
+        end
+
+        local loc = token.loc
+        local end_loc = token.end_loc
+        local value = ""
+        if token.value then
+            local s = token.value
+            if type(token.value) == "string" then
+                s = C.string(token.value)
+            end
+            value = string.format(" %s", s)
+        end
```

Comment:
Suggestion: 

```
local value
if token.value then
    if type(token.value) == "string" then
        value = " " .. C.string(token.value)
    else
        value = " " .. tostring(token.value)
    end
else
    value = ""
end
```

I think an if with an else is easier to understand what is going on than an if without an else, where the value of the variable is overwritten.